frontend/src/router/pages/SubmitDates.jsx

```diff
@@ -101,6 +101,16 @@ function SubmitDates() {
     return notes;
   }
 
+  function continueToPreviewEnabled() {
+    const allDatesHaveValues = Object.values(dates).some((dateType) =>
```

Comment:
Even though this array will only have one item (I think?), we should check with `.every` instead of `.some`
```suggestion
    const allDatesHaveValues = Object.values(dates).every((dateType) =>
```